### PR TITLE
Task #5233 - Manage category

### PR DESCRIPTION
**Pull request by taylq**
https://edu-redmine.framgia.vn/issues/5233

https://edu-redmine.framgia.vn/issues/5232

https://edu-redmine.framgia.vn/issues/5231

https://edu-redmine.framgia.vn/issues/5230

https://edu-redmine.framgia.vn/issues/5234

### CODE REVIEW

**Comment by taylq** [comment]
ready

File: app/views/admin/categories/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,71 @@
+<div class="right_col" role="main">
+  <div class="">
+    <div class="page-title">
+      <%= render 'layouts/admin/flash' %>
```
Comment (huuphuc93) [inline app/views/admin/categories/index.html.erb]: dùng " thay cho '

**Comment by huuphuc93** [comment]
commented

**Comment by taylq** [comment]
ready

**Pull request merged**